tensorflow/lite/interpreter.cc

```diff
@@ -247,6 +247,20 @@ TfLiteStatus Interpreter::SetExecutionPlan(const std::vector<int>& new_plan) {
 void Interpreter::UseNNAPI(bool enable) { primary_subgraph().UseNNAPI(enable); }
 
 void Interpreter::SetNumThreads(int num_threads) {
+  // num_threads should be >= -1.
```

Comment:
As these comments have been put in the header file where this function is declared, let's just remove them here?